components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/Utils.java

```diff
@@ -41,16 +40,16 @@ public static String getQueryParameter(MessageContext messageContext, String key
         return null;
     }
 
-    public static void setJsonPayLoad(org.apache.axis2.context.MessageContext axis2MessageContext, JSONObject payload){
+    public static void setJsonPayLoad(org.apache.axis2.context.MessageContext axis2MessageContext, String payload){
```

Comment:
Is changing the argument type necessary? This will require us to call the toString of the JSON object every-time we invoke this method

Reply:
Fixed.